setup.py

```diff
@@ -38,11 +35,6 @@ def run(self):
         _sdist.run(self)
 
 
-class install(_install):
```

Comment:
Unfortunately we're installing via a setup.py and dependency_links - I've not found a way of specifying the equivalent of `-e` in that situation.

i.e.
```python
<snip>
dependency_links=['git+https://github.com/crbunney/uap-python.git@fix_pip_install_from_source#egg=ua-parser-0.4.0'],  # Run pip install with --process-dependency-links flag to use dependency_links
install_requires=['ua-parser==0.4.0'],
<snip>
```
because we wanted to use some code that wasn't yet on pypi.

Hence the unusual use case